### PR TITLE
add spacevim

### PR DESCRIPTION
**Pull request by wsdjeg**
close https://github.com/neurosnap/neovimcraft/pull/296

### CODE REVIEW

File: data/resources-config.json [hunk 1]
```diff
@@ -1275,6 +1275,21 @@
...
+    {
+      "type": "github",
+      "username": "SpaceVim",
+      "repo": "SpaceVim",
+      "tags": [
+        "spacevim",
+        "distribution",
+        "neovim-config",
+        "neovim-configuration",
+        "neovim-plugin",
```
Comment (neurosnap) [inline data/resources-config.json:1287]: Please review the available tags and find the ones that match.

**Review by neurosnap — CHANGES_REQUESTED** [review]

File: data/resources-config.json (hunk 1 above)
Comment (wsdjeg) [reply data/resources-config.json]: I have updated the tags of spacevim, thanks.

File: data/resources-config.json [hunk 2]
```diff
@@ -1275,6 +1275,16 @@
         "zshrc"
       ]
     },
+    {
+      "type": "github",
+      "username": "SpaceVim",
+      "repo": "SpaceVim",
+      "tags": [
+        "vim",
```
Comment (neurosnap) [inline data/resources-config.json:1283]: Delete

File: data/resources-config.json [hunk 3]
```diff
@@ -1275,6 +1275,16 @@
...
+        "nvim",
```
Comment (neurosnap) [inline data/resources-config.json:1284]: Delete

File: data/resources-config.json [hunk 4]
```diff
@@ -1275,6 +1275,16 @@
...
+        "plugins"
```
Comment (neurosnap) [inline data/resources-config.json:1285]: Add preconfigured-configuration

**Review by neurosnap — CHANGES_REQUESTED** [review]

**Review by neurosnap — APPROVED** [review]

**Comment by wsdjeg** [comment]
@neurosnap Hi, since this PR is merged, but still can not found spacevim in https://neovimcraft.com/

**Comment by neurosnap** [comment]
Hmm might be a bug, need to investigate

**Pull request merged**